### PR TITLE
Pagination time

### LINKED ISSUE #1729: Campaign Finder - "show more" pagination
Include a "Show More" button if there are more campaign results, when the user clicks it should populate an additional 12 campaigns.

### PR DESCRIPTION
**Pull request by DFurnes**
:page_with_curl: :page_with_curl: :page_with_curl: :page_with_curl: :page_with_curl: :page_with_curl: 

Fixes #1729.

### CODE REVIEW

**Comment by weerd** [comment]
I believe it looks good :+1:

File: lib/themes/dosomething/paraneue_dosomething/js/finder/ResultsView.js [hunk 1]
```diff
@@ -18,17 +18,25 @@ define(["jquery", "unveil", "finder/Campaign"], function($, unveil, Campaign) {
...
      * @param  {jQuery} $div The div where the results will live
+     * @param  {jQuery} $blankSlateDiv The div containing initial pre-filtered state
+     * @param  {function} paginateAction Function to query next page of results.
      */
-    init: function ($div, $blankSlateDiv) {
+    init: function ($div, $blankSlateDiv, paginateAction) {
       ResultsView.$div = $div;
       ResultsView.$div.hide();
 
+      ResultsView.$paginationLink = $("<div class='pagination-link'><a href='#' class='secondary js-finder-show-more'>Show More</a></div>");
```
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/js/finder/ResultsView.js:34]: No use of template? Probably not needed since it's such a small piece.

Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/js/finder/ResultsView.js:34]: Definitely. I'd like to take a second pass and refactor the ResultsView to render into a container built from a template. But let's hold off on that until after launch.

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/js/finder/ResultsView.js:34]: :dancers:

**Pull request merged**